app/NavEntry.js

```diff
@@ -29,6 +34,27 @@ class MainNavigation extends React.Component {
     }
   }
 
+  async componentDidMount() {
+    const { t } = this.props;
+    try {
+      const { data } = await fetch(BULLETINS_URL);
+      const bulletins = data.bulletins;
+      const lastBulletin = await GetStoreData('lastBulletin', true);
+      console.log(lastBulletin);
```

Comment:
What this console log for??? please delete it

Reply:
Ups